python/openmldb/dbapi/dbapi.py

```diff
@@ -327,9 +343,103 @@ def execute(self, operation, parameters=()):
         else:
             raise DatabaseError("unsupport sql")
 
+    def __insert_rows(self, rows: List[Union[tuple, dict]], hold_idxs, schema, rows_builder, command):
+        for row in rows:
+            tmp_builder = rows_builder.NewRow()
```

Comment:
building one row is the same with https://github.com/4paradigm/OpenMLDB/blob/060120110a76a11806eb380fb5289a5e88ff7ab7/python/openmldb/dbapi/dbapi.py#L207-L220

reuse it. And you need to merge main, this file is changed.